### DISCUSSION TITLE
AzuraRelay: AzuraCast's Lightweight, Automated Relay Companion

### DISCUSSION THREAD

**Discussion by Marcioghiraldelli** [AzuraCast/AzuraCast#6692] [General]
I have a doubt about this tool.
I saw that it does an automatic relay from the main server, even copying the ports from the main server, this is very cool, but my question is, if I broadcast live, will the live broadcast be processed on the relay server or on the main server, if so? be it on the main, is there a possibility that the live one will be processed on the relay instead of the main one?

**Comment by BusterNeece** [comment]
It relays the final broadcast as it's sent out to the mount points, so the live DJs connecting in are done on the AutoDJ side and the combined signal is relayed out by AzuraRelay.
